gallery/plot_visualization_utils.py

```diff
@@ -68,7 +67,9 @@ def show(imgs):
 # models.  Here is demo with a Faster R-CNN model loaded from
 # :func:`~torchvision.models.detection.fasterrcnn_resnet50_fpn`
 # model. You can also try using a RetinaNet with
-# :func:`~torchvision.models.detection.retinanet_resnet50_fpn`. For more details
+# :func:`~torchvision.models.detection.retinanet_resnet50_fpn`, an SSDlite with
+# :func:`~torchvision.models.detection.ssdlite320_mobilenet_v3_large` or an SSD with
+# :func:`~torchvision.models.detection.ssd300_vgg16`. For more details
```

Comment:
Happy to undo this. I'm not sure what criteria we used to add models here but clearly does not scale to add all of them.

Reply:
that's fine, once it becomes intractable we can think of a more holistic approach, but it doesn't overload the example for now